inst/examples/census/config.yml

```diff
@@ -24,9 +23,8 @@ cloudml:
   eval_file                : "gs://rstudio-cloudml-demo-ml/census/data/adult.test"
   train_file               : "gs://rstudio-cloudml-demo-ml/census/data/adult.data"
 
-  # Outputs (to run within another gcloud account, substitute your own buckets)
-  job_dir                  : !expr cloudml::unique_job_dir("gs://rstudio-cloudml-demo-ml/census/jobs")
-  staging_bucket           : "gs://rstudio-cloudml-demo-ml"
```

Comment:
It looks like after this change we no longer specify a staging bucket, is that okay?